client/src/pages/organizations/Form.js

```diff
@@ -690,13 +690,18 @@ const BaseOrganizationForm = ({ currentUser, edit, title, initialValues }) => {
       new Organization(values),
       "notes",
       "childrenOrgs",
-      "positions"
+      "positions",
+      "tasks"
+    )
+    // strip tasks fields not in data model
+    organization.tasks = values.tasks.map(t =>
+      Object.without(t, "formCustomFields")
```

Comment:
Can we use `utils.getReference` instead here?

Reply:
I don't see how